services/stsafea/stsafea_sessions.c

```diff
@@ -526,7 +527,8 @@ static stse_ReturnCode_t stsafea_session_frame_r_mac_verify( stse_session_t *pSe
 			}
 		}
 
-		ret = stse_platform_aes_cmac_verify_finish(pMAC);
+		memcpy(aes_cmac_block, pMAC, STSAFEA_MAC_SIZE);
```

Comment:
Could you provide more info about this change ?

Reply:
No, this is to allow stse_platform_aes_cmac_verify_finish to work on a buffer that has the size of an AES block (that would be the max possible size of an STSAFE MAC). Then, inside the stse_platform_aes_cmac_verify_finish, you compute the MAC and you only compare the bytes relevant for the STSAFE (4 bytes for STSAFEA_MAC_SIZE).